examples/kitchensink-ts/README.md

```diff
@@ -0,0 +1,3 @@
+# Strapi application
+
+A quick description of your strapi application
```

Comment:
What about : `Strapi application generated in TypeScript` ?


We could add something like : 

```
Installed plugins:
- pluginA,
- pluginB... ?

### Customisations:

No customisation
```

Reply:
Yes, we could, I'm not even sure about keeping the name `kitchensink-ts` (same for the structure), it was more an example to test the PR than anything else, really
I'll change the title for the moment